pensieve/statistics.py

```diff
@@ -0,0 +1,160 @@
+import attr
+import cattr
+import mozanalysis.bayesian_stats.bayesian_bootstrap as mabsbb
+from google.cloud import bigquery
+from google.api_core.exceptions import NotFound
+from typing import Callable, Any, Dict, List, Tuple, Optional
+from pandas import DataFrame
+import pandas
+import numpy as np
+
+from pensieve.pre_treatment import PreTreatment, RemoveNulls
+
+
+@attr.s(auto_attribs=True)
+class StatisticResult:
+    """
+    Represents the resulting data after applying a statistic transformation
+    to metric data.
+    """
+
+    metric: str
+    statistic: str
+    parameter: float
+    label: str
+    ci_width: Optional[float] = 0.0
+    point: Optional[float] = 0.0
+    lower: Optional[float] = 0.0
+    upper: Optional[float] = 0.0
+
+    def with_ci(self, data: DataFrame, t: float, confidence_level: float) -> "StatisticResult":
```

Comment:
I'm not entirely sure if that is how the CI is calculated.

Reply:
In general, no. The statistic needs to provide this.